integr8ly/cloud-resource-operator
# allow extra secret keys to be set for openshift blob storage

for openshift blob storage we assume the same structure as aws. in
some scenarios such as using minio, extra keys may need to be used
to specify a full aws s3-compatible storage.

this change allows for additional secret keys to be added to the
resulting secret when using the openshift proivder. the operator
will not override these values and will not remove them.

verification:
- run the operator against a cluster
- create a blobstorage using the openshift provier
- update the result secret with extra keys
- ensure the keys are not removed when the operator reconciles
